src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs

```diff
@@ -42,7 +42,7 @@ private static Listener EnlistTarget(object target, Func<string, bool> isEnabled
             var methodInfos = typeInfo.DeclaredMethods;
             foreach (var methodInfo in methodInfos)
             {
-                var notificationNameAttribute = methodInfo.GetCustomAttribute<TelemetryNameAttribute>();
+                var notificationNameAttribute = methodInfo.GetCustomAttribute<DiagnosticNameAttribute>();
```

Comment:
I see, this is the only instance of it and I didn't know that bit of the history.